### DISCUSSION TITLE
Support for tvOS

### DISCUSSION THREAD

**Discussion by 8kdesign** [sbooth/SFBAudioEngine#287] [Ideas]
Hi! 

Thanks for creating SFBAudioEngine.  Love the metadata editing functionality.
Are there any plans to support tvOS?

Kind regards,
Wen Hao

**Comment by sbooth** [comment]
I don't have any plans to support tvOS *per se* but I also am not opposed to it. I think a good first step would be to attempt to build the XCFrameworks for tvOS and see what happens.

**Comment by sbooth** [comment]
I have a version that compiles for tvOS at https://github.com/sbooth/SFBAudioEngine/tree/tvos

It's mostly untested at this point.

**Reply by 8kdesign** [reply]
Nice! Thanks

**Comment by sbooth** [comment]
As of a001534860677a00affddd8952c4198160cf7153 tvOS support has been added.
